### PR TITLE
Update information for el8 foreman packages

### PR DESCRIPTION
**Pull request by zjhuntin**

### CODE REVIEW

File: rel-eng/tito.props [hunk 1]
```diff
@@ -347,6 +347,311 @@ whitelist = foreman
   tfm
 scl = tfm
 
+[foreman-nightly-rhel8]
+disttag = .el8
+whitelist = foreman
```
Comment (ekohl) [inline rel-eng/tito.props]: Do we want to start the tag empty and fill the whitelist in follow up PRs?

File: rel-eng/tito.props [hunk 2]
```diff
@@ -347,6 +347,311 @@ whitelist = foreman
...
+  rubygem-validates_lengths_from_database
+  rubygem-webpack-rails
+  rubygem-websocket-driver
+  rubygem-websocket-extensions
+  rubygem-will_paginate
+  rubygem-wirb
+  rubygem-x-editable-rails
+  rubygem-xmlrpc
+  tfm
+scl = tfm
```
Comment (ekohl) [inline rel-eng/tito.props]: Don't think this should be here

File: rel-eng/tito.props [hunk 3]
```diff
@@ -347,6 +347,311 @@ whitelist = foreman
...
+  rubygem-useragent
+  rubygem-validates_lengths_from_database
+  rubygem-webpack-rails
+  rubygem-websocket-driver
+  rubygem-websocket-extensions
+  rubygem-will_paginate
+  rubygem-wirb
+  rubygem-x-editable-rails
+  rubygem-xmlrpc
+  tfm
```
Comment (ekohl) [inline rel-eng/tito.props]: Don't think this should be here

**Comment by zjhuntin** [comment]
[test rpm]

**Comment by zjhuntin** [comment]
[test rpm]

**Comment by zjhuntin** [comment]
[test rpm]

**Comment by zjhuntin** [comment]
[test rpm]

**Comment by zjhuntin** [comment]
[test rpm]

File: package_manifest.yaml [hunk 4]
```diff
@@ -24,13 +24,16 @@ foreman_scl_packages:
...
     releasers:
       - koji-foreman
     nightly_releaser: koji-foreman-jenkins
-    repoclosure_config: repoclosure/yum_el7.conf
+    repoclosure_config: repoclosure/yum.conf
     repoclosure_target_repos:
       rhel7:
         - el7-foreman-nightly
+      rhel8:
+        - foreman-nightly-el8
```
Comment (pcreech) [inline package_manifest.yaml]: this one should be el8-foreman-nightly, apologies for telling you the wrong thing.  This is based on what the repoid is in yum.conf `[el8-foreman-nightly]`

File: package_manifest.yaml [hunk 5]
```diff
@@ -39,6 +42,12 @@ foreman_scl_packages:
         - el7-extras
         - el7-puppet-6
         - el7-scl
+      rhel8:
```
Comment (pcreech) [inline package_manifest.yaml]: So, based on the error I see in the jenkins job, it appears it's looking for a key of `el8`, not `rhel8`, perhaps we can try that to see if it works?  For here and for above

File: package_manifest.yaml [hunk 6]
```diff
@@ -24,13 +24,16 @@ foreman_scl_packages:
...
+      - foreman-nightly-rhel8
     releasers:
       - koji-foreman
     nightly_releaser: koji-foreman-jenkins
-    repoclosure_config: repoclosure/yum_el7.conf
+    repoclosure_config: repoclosure/yum.conf
     repoclosure_target_repos:
       rhel7:
         - el7-foreman-nightly
+      rhel8:
```
Comment (pcreech) [inline package_manifest.yaml]: As per the other comment, perhaps htis needs to be `el8` instead of `rhel8`

File: repoclosure/yum.conf [hunk 7]
```diff
@@ -78,6 +78,10 @@ baseurl=https://yum.puppetlabs.com/puppet6/el/7/x86_64/
 name=Foreman nightly EL7
 baseurl=http://koji.katello.org/releases/yum/foreman-nightly/RHEL/7/x86_64/
 
+[el8-foreman-nightly]
+name=Foreman nightly EL8
+baseurl=http://koji.katello.org/releases/yum/foreman-nightly/RHEL/8/x86_64/
```
Comment (pcreech) [inline repoclosure/yum.conf]: This repo doesn't yet exist, so we may have a minor chicken-egg issue here.

**Comment by pcreech** [comment]
The repoclosure failure is expected, per the comment above with this repo not existing yet:

`http://koji.katello.org/releases/yum/foreman-nightly/RHEL/8/x86_64/repodata/repomd.xml: [Errno 14] HTTP Error 404 - Not Found`

**Comment by zjhuntin** [comment]
[test rpm]

**Comment by zjhuntin** [comment]
[test rpm]

**Comment by ehelms** [comment]
[test rpm]

File: rel-eng/tito.props [hunk 8]
```diff
@@ -347,6 +347,10 @@ whitelist = foreman
   tfm
 scl = tfm
 
+[foreman-nightly-rhel8]
+disttag = .el8
+whitelist = rubygem-concurrent-ruby
```
Comment (ekohl) [inline rel-eng/tito.props]: Bit odd to add this to the whitelist, but not the comps.

**Pull request merged**